### PR TITLE
#990: Random Taco

### LINKED ISSUE #990: Taco Randomizer Command
https://github.com/evz/tacofancy-api

### PR DESCRIPTION
**Pull request by merelyAnna**
Closes #990: Added taco-command returning a random taco consisting of base layer, mixin, condiment, seasoning and shell.

### CODE REVIEW

**Comment by davipatury** [comment]
*Tip:* You can use `Closes #990` instead of `Fix for #990` so the issue gets closed as soon as this PR gets merged.

**Comment by merelyAnna** [comment]
@davipatury Thanks, I didn't know. I'll edit the PR :)

File: src/commands/misc/taco.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    })
+  }
+
+  async run ({ t, author, channel }) {
+    const embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+    const body = await fetch(
+      'http://taco-randomizer.herokuapp.com/random'
+    ).then(res => res.json())
+    const randomTaco = `${body.base_layer.name} with ${body.mixin.name} garnished with ${body.seasoning.name} and topped of with ${body.condiment.name} with and wrapped in ${body.shell.name}`
```
Comment (pedrofracassi) [inline src/commands/misc/taco.js]: This should be translatable.

Comment (merelyAnna) [reply src/commands/misc/taco.js]: The hard coded parts of the string are certainly translatable via i18n. The ingredients however would need to be put through a translator. I saw you have some i18n bit in the code, but I'm not sure about a translating API or something. If you can point me in the right direction, I'm happy to fix this.

Comment (Doges) [reply src/commands/misc/taco.js]: > The hard coded parts of the string are certainly translatable via i18n. The ingredients however would need to be put through a translator. I saw you have some i18n bit in the code, but I'm not sure about a translating API or something. If you can point me in the right direction, I'm happy to fix this.

that actually makes sense, its weird something like "Lettuce com Oion e um pouco de..."

Comment (merelyAnna) [reply src/commands/misc/taco.js]: So... you want to change anything here?

Comment (metehus) [reply src/commands/misc/taco.js]: well on their repo theres a .db file so its gonna be kinda hard to get all i think and it should be updated in a while so whats your deal @pedrofracassi

**Comment by pedrofracassi** [comment]
I'm pretty sure the ingredients are user submitted, so that will make it even harder to translate

**Comment by metehus** [comment]
> I'm pretty sure the ingredients are user submitted, so that will make it even harder to translate

google translate all?

**Pull request closed without merging**